partner_utm_source/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    source_id = fields.Many2one("utm.source", string="Source")
```

Comment:
Partner is a very connected model, I think maybe prefixing this field with "utm_", specify UTM source also in string or help attribute would be good. I did check for collisions in CE full demo instance and nothing appeared but ... ¿What do U think? ¿Too cautious ...?

Reply:
Thanks for your review, I agree that prefixing the field with "utm_" would make it clearer